src/components/quality-page/quality-content-group.styles.scss

```diff
@@ -0,0 +1,38 @@
+.top-section {
+    display: flex;
+}
+
+.number {
+    font-size: 5.5em;
+    font-family: 'Times New Roman', Times, serif;
+    font-style: italic;
+    color: #BFB4AD;
+}
+
+.subheading {
+    font-size: 1.75em;
+    font-family: YuMincho;
+    font-style: italic;
+    color: #5A4537;
+    width: 22vw;
+    line-height: 6.6vh;
```

Comment:
line height shouldnt use vh.
its probably best to almost never use vh/vw with text, unless its purely decorative.
if you change the height of the browser it squishes together and lines may even overlap.
you can use px/em/rem for line height.

in fact, in all cases, you should try not to use vh/vw unless you see no other alternative.

Reply:
That was just a mistake I forgot to change last time, changed to line height.